polkadot/cli/src/cli.rs

```diff
@@ -166,6 +166,11 @@ pub struct RunCmd {
 	///  **Dangerous!** Do not touch unless explicitly advised to.
 	#[arg(long, hide = true)]
 	pub collator_protocol_hold_off: Option<u64>,
+
+	/// Enable or disable per validator collected approvals metrics
+	/// to be published to prometheus. If not specified, set to false.
+	#[arg(long)]
+	pub verbose_approval_metrics: Option<bool>,
```

Comment:
Why does this need to be an `Option<bool>`? Could it not just be an `bool` which defaults to `false`?